test/run/test_sources.py

```diff
@@ -34,9 +34,9 @@ def netns():
     with open("/proc/self/ns/net") as oldnet:
         # Create a new namespace and enter it.
         libc.unshare(CLONE_NEWNET)
```

Comment:
This should probably also go into the `try` block. An async exception after the unshare would thus leave us in a new namespace again. But then again, async exceptions are broken in python, anyway.

Reply:
My reasoning for voting to not include it was that if the first unshare fails we technically shouldn't do the setns one in the finally block. Otoh this will hopefully not fail and just be a no-op.